### PR TITLE
Fix cpu-mode build and minors in docs

### PR DESCRIPTION
**Pull request by LiSu**
1. fix cpu-mode build 
2. minors in docs

### CODE REVIEW

File: examples/igbh/README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Note that in `dataset.py`, we have converted the graph into an undirected graph.
 
 ## 2. Single node and single GPU training:
 ```
-python train_rgnn.py --dataset_size='tiny' --model='gat' --data_sze='tiny'--num_classes=19
+python train_rgnn.py --dataset_size='tiny' --model='rgat' --dataset_size='tiny' --num_classes=19
```
Comment (Seventeen17) [inline examples/igbh/README.md:35]: Duplicated `dataset_size`

File: README.md [hunk 2]
```diff
@@ -145,6 +145,9 @@ sh install_dependencies.sh
 python setup.py bdist_wheel
 pip install dist/*
 ```
+To build in CPU-only mode, set 'WITH_CUDA' to 'OFF' in [CMakeLists.txt](./CMakeLists.txt) 
+and [setup.py](./setup.py).
```
Comment (Seventeen17) [inline README.md:149]: We should not ask users to change the file. Actually changing the command to `WITH_CUDA=OFF  python setup.py bdist_wheel` works.

Comment (LiSu) [reply README.md]: fixed

File: examples/igbh/README.md (hunk 1 above)
Comment (LiSu) [reply examples/igbh/README.md]: fixed

**Review by Seventeen17 — APPROVED** [review]

**Review by Seventeen17 — APPROVED** [review]

**Pull request merged**